src/libs/UserService/index.ts

```diff
@@ -1,3 +1,17 @@
 import { Users } from '../../database/models';
 
-export const getUser = async (id) => await Users.findByPk(id).then(result => result.get({plain: true}))
\ No newline at end of file
+export const getUser = async (id) => await Users.findByPk(id).then(result => result.get({plain: true}))
+
+export const getUserFirstAndLastName = async (id) => {
+    const first_name = await Users.findAll({
+        where: { id },
+    }).then(result => result[0].first_name)
```

Comment:
```suggestion
    const first_name = await Users.findByPk(id),.then(result => result.first_name)
```
Since we are only returning 1 result and we are fetching by primary key, The findByPk() function should help shorten the code here a bit